chef/data_bags/crowbar/bc-template-ceph.json

```diff
@@ -6,7 +6,9 @@
       "master": false,
       "disk_mode": "all",
       "config": {
-        "fsid" : ""
+        "fsid" : "",
+        "osds_in_total": 0,
```

Comment:
(replying to the thread at https://github.com/crowbar/barclamp-ceph/pull/92#discussion_r26652178)

So I guess I'm really missing something: what's the use case where people will want to set a value here instead of relying on the automatic calculation? I still fail to see why we need this as an attribute.